docs/get-started/getting-started-gcp.md

```diff
@@ -272,17 +264,16 @@ that prepares the dataset for training.  The ETL query splits the data, saving 2
 a separate GCS location training for evaluation.  Using the default notebook configuration the
 first stage should take ~110 seconds (1/3 of CPU execution time with same config) and the second
 stage takes ~170 seconds (1/7 of CPU execution time with same config).  The notebook depends on
-the pre-compiled [Spark RAPIDS SQL
-plugin](https://mvnrepository.com/artifact/com.nvidia/rapids-4-spark) which is pre-downloaded by
-the GCP Dataproc [RAPIDS init
-script](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/rapids).
+the pre-compiled [RAPIDS plugin for Apache Spark](https://mvnrepository.com/artifact/com.nvidia/rapids-4-spark) which is pre-downloaded and pre-configured by
+the GCP Dataproc [Spark RAPIDS init
+script](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/spark-rapids).
```

Comment:
Line wrap and RAPIDS Accelerator instead of Spark RAPIDS
```suggestion
the pre-compiled [RAPIDS Accelerator for Apache Spark](https://mvnrepository.com/artifact/com.nvidia/rapids-4-spark)
which is pre-downloaded and pre-configured by the GCP Dataproc
[RAPIDS Accelerator init script](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/spark-rapids).
```